bin/fetch-configlet

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
```

Comment:
A good rule for PRs is to have them be as specific as possible. This PR is about adding an exercise, which means that ideally the changes to this file should not be part of this PR. Could you revert these changes to this file?